ar3:
    hardware_interface:
        loop_hz: 10 # hz
        joints:
            - joint_1
            - joint_2
            - joint_3
            - joint_4
            - joint_5
            - joint_6